virttest/utils_test/libvirt.py

```diff
@@ -488,7 +509,6 @@ def setup_or_cleanup_iscsi(is_setup, is_login=True,
         _iscsi.emulated_id = _iscsi.get_target_id()
         _iscsi.cleanup()
         process.run("rm -f %s" % emulated_path)
-        process.run("vgscan --cache", ignore_status=True)
```

Comment:
I am not sure why we use this command here, but it really caused some problems.
2016-04-15 01:47:28,988 process          L0331 INFO | Running 'vgscan --cache'
2016-04-15 01:47:28,993 process          L0418 DEBUG| [stderr] File descriptor 3 (/root/avocado/job-results/job-2016-04-15T01.46-bb44a47/job.log) leaked on vgscan invocation. Parent PID 6794: python
2016-04-15 01:47:28,994 process          L0418 DEBUG| [stderr] File descriptor 4 (pipe:[388333]) leaked on vgscan invocation. Parent PID 6794: python
2016-04-15 01:47:28,994 process          L0418 DEBUG| [stderr] File descriptor 5 (/dev/urandom) leaked on vgscan invocation. Parent PID 6794: python
2016-04-15 01:47:28,994 process          L0418 DEBUG| [stderr] File descriptor 6 (pipe:[388333]) leaked on vgscan invocation. Parent PID 6794: python
2016-04-15 01:47:28,994 process          L0418 DEBUG| [stderr] File descriptor 7 (/dev/null) leaked on vgscan invocation. Parent PID 6794: python
2016-04-15 01:47:28,994 process          L0418 DEBUG| [stderr] File descriptor 8 (/dev/urandom) leaked on vgscan invocation. Parent PID 6794: python
2016-04-15 01:47:28,994 process          L0418 DEBUG| [stderr] File descriptor 9 (/dev/urandom) leaked on vgscan invocation. Parent PID 6794: python